homeassistant/config_entries.py

```diff
@@ -2221,7 +2211,7 @@ async def async_wait_component(self, entry: ConfigEntry) -> bool:
         # The component was not loaded.
         if entry.domain not in self.hass.config.components:
             return False
-        return entry.state == ConfigEntryState.LOADED
+        return entry.state is ConfigEntryState.LOADED
```

Comment:
Why is this changed?

Reply:
All of these should be identity checks since enums are singletons. This one was missed in earlier cleanups